checkmarx/util.py

```diff
@@ -481,3 +484,39 @@ def get_checkmarx_cfg(checkmarx_cfg_name: str) -> cmmmodel.CheckmarxConfig:
 @functools.lru_cache()
 def create_checkmarx_client(checkmarx_cfg: cmmmodel.CheckmarxConfig):
     return checkmarx.client.CheckmarxClient(checkmarx_cfg)
+
+
+def upload_results_to_delivery_service(
```

Comment:
code organisation could still be improved:

- have a function that will create artefactmetadata from scan-result
- call this function in template, where scan-results are present

instrumenting delivery-service-client is not specific to the type of uploaded data, thus such code should not (redundantly) be placed in packages specific to certain metadata types.